mim/commands/download.py

```diff
@@ -4,7 +4,9 @@
 
 import click
 
-from mim.click import OptionEatAll, get_downstream_package, param2lowercase
+from mim.click import OptionEatAll
+from mim.click import argument as compat_argument
```

Comment:
We can directly use argument rather than compat_argument.

Reply:
Done